code/datums/weather/weather.dm

```diff
@@ -105,15 +105,16 @@
 	weather_duration = rand(weather_duration_lower, weather_duration_upper)
 	SSweather.processing |= src
 	update_areas()
-	for(var/mob/M AS in GLOB.player_list)
-		var/turf/mob_turf = get_turf(M)
-		if(mob_turf && (mob_turf.z in impacted_z_levels))
-			if(telegraph_message)
-				to_chat(M, telegraph_message)
-			if(M?.client?.prefs?.toggles_sound & SOUND_WEATHER)
-				continue
-			if(telegraph_sound)
-				SEND_SOUND(M, sound(telegraph_sound, volume = 60))
+	for(var/mob/impacted_mob AS in GLOB.player_list)
+		if(impacted_mob?.client?.prefs?.toggles_sound & SOUND_WEATHER)
+			continue
```

Comment:
the sound pref now disabled the message as well.

Reply:
Ah fucks sake